src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.Array.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
+        /// <summary>
+        ///   Reads the next data item as the start of an array (major type 4)
+        /// </summary>
+        /// <returns>
+        ///   The length of the definite-length array, or <c>null</c> if the array is indefinite-length.
+        /// </returns>
+        /// <exception cref="InvalidOperationException">
+        ///   the next data item does not have the correct major type.
+        /// </exception>
+        /// <exception cref="FormatException">
+        ///   invalid CBOR encoding data --OR--
```

Comment:
I think you got this from me, but I just double checked, and apparently it's supposed to be `-or-` (one hyphen each side, not two, and lowercase, not uppercase).  Guess I get to push that commit to AsnReader now :)

```suggestion
        ///   invalid CBOR encoding data -or-
```

Reply:
It might even be more complicated, in the XML it's also preceded and succeeded by a blank line.

https://github.com/dotnet/dotnet-api-docs/blob/8e6f154322350a3aa7bb5858891930ed50454ba0/xml/System.Security.Principal/NTAccount.xml#L83-L87

There's a separate process for copy things from here to the docs site, but the more conforming things are here the easier it is to copy them over.

Reply:
I see lots of variations of `-or-` in the wild, but no documentation. @carlossanlop what say you?

Reply:
Hey @eiriktsarpalis I just saw you mentioned me here. Jeremy's comments show the correct formatting.
```
Foo bar text.

-or-

Foo bar text other.
```